### PR TITLE
Fix bad character f5 vs and pool

### PR DESCRIPTION
**Pull request by cgagnaire**
Fix bad first character in long and short outputs of virtual servers and pools name of F5 BigIP plugin.

### CODE REVIEW

File: network/f5/bigip/snmp/mode/poolstatus.pm [hunk 1]
```diff
@@ -197,8 +196,9 @@ sub manage_selection {
         my $result = $options{snmp}->map_instance(mapping => $mapping->{$map}, results => $snmp_result->{$branch_name}, instance => $instance);
         $result->{Name} = '';
         foreach (split /\./, $instance) {
-            $result->{Name} .= chr  if ($_ >= 32 && $_ <= 126);
+            $result->{Name} .= chr if ($_ >= 32 && $_ <= 126);
         }
+        $result->{Name} =~ s/^.//;
```
Comment (Sims24) [inline network/f5/bigip/snmp/mode/poolstatus.pm]: Looks really odd to fix it this way.   Do we have on the root cause of this first random char ? Is this chr provoking it

File: network/f5/bigip/snmp/mode/virtualserverstatus.pm [hunk 2]
```diff
@@ -187,8 +186,10 @@ sub manage_selection {
         
         $result->{Name} = '';
         foreach (split /\./, $instance) {
-            $result->{Name} .= chr  if ($_ >= 32 && $_ <= 126);
+            $result->{Name} .= chr if ($_ >= 32 && $_ <= 126);
         }
+        $result->{Name} =~ s/^.//;
```
Comment (Sims24) [inline network/f5/bigip/snmp/mode/virtualserverstatus.pm]: See comment on poolstatus 

Thanks

File: network/f5/bigip/snmp/mode/poolstatus.pm (hunk 1 above)
Comment (cgagnaire) [reply network/f5/bigip/snmp/mode/poolstatus.pm]: Not that odd. The names are built from oid indexes that are ASCII translation of the names. But there always is a random charactere in first position.

Comment (Sims24) [reply network/f5/bigip/snmp/mode/poolstatus.pm]: roger that, just wonder why this char is popping ... let's merge to let everybody get access to this fix

Comment (cgagnaire) [reply network/f5/bigip/snmp/mode/poolstatus.pm]: Yea I know, hope its not a bug from my f5 box...

Comment (Sims24) [reply network/f5/bigip/snmp/mode/poolstatus.pm]: Need to test it against our snmp simulator on friday. we got one walk from other user ;)

Comment (cgagnaire) [reply network/f5/bigip/snmp/mode/poolstatus.pm]: Already tested, and there is no pools or virtualservers :/

**Pull request merged**